tests/test_create.py

```diff
@@ -54,6 +54,16 @@ def test_create_stage(sql_compiler):
     )
     assert actual == expected
 
+    create_stage = CreateStage(stage=stage, container=container, temporary=True)
+    # validate that the resulting SQL is as expected
+    actual = sql_compiler(create_stage_replace)
```

Comment:
I think this one should be:
```suggestion
    actual = sql_compiler(create_stage)
```